### PR TITLE
🐛 passes a ref that allows term agg to stay in view on 'less'

### PR DESCRIPTION
**Pull request by jgnieuwhof**
depends on https://github.com/overture-stack/arranger/pull/363

### CODE REVIEW

File: src/components/FileRepo.js [hunk 1]
```diff
@@ -199,7 +201,7 @@ const AggregationsWrapper = enhance(({ state, effects, theme, setSQON, ...props
         />
         <UploadIdsButton {...{ theme, effects, state, setSQON, ...props }} />
       </div>
-      <Aggregations {...{ ...props, setSQON }} />
+      <Aggregations {...{ ...props, setSQON, containerRef: aggregationsWrapperRef }} />
```
Comment (hlminh2000) [inline src/components/FileRepo.js]: YES!!! React 16.3 rocks!!!!

File: src/components/FileRepo.js [hunk 2]
```diff
@@ -125,9 +125,11 @@ const UploadIdsButton = ({ theme, state, effects, setSQON, ...props }) => (
   </div>
 );
 
+const aggregationsWrapperRef = React.createRef();
```
Comment (hlminh2000) [inline src/components/FileRepo.js]: YES!!! React 16.3 rocks!!!!

Comment (hlminh2000) [reply src/components/FileRepo.js]: woops! maybe declare this inside `AggregationsWrapper` so we don't have a singleton ref :P

**Comment by jgnieuwhof** [comment]
@hlminh2000 - addressed the singleton issue :)

**Pull request merged**